CMake/doxyfile.in

```diff
@@ -134,7 +134,7 @@ INPUT                  += @Viskores_SOURCE_DIR@/docs/CodingConventions.md
 INPUT                  += @Viskores_SOURCE_DIR@/viskores
 INPUT                  += @Viskores_BINARY_DIR@/include
 
-USE_MDFILE_AS_MAINPAGE = README.md
+USE_MDFILE_AS_MAINPAGE = @Viskores_SOURCE_DIR@/RREADME.md
```

Comment:
`README` is misspelled.

Also, is it really necessary to include `README.md` in the doxygen generation? I always get many warnings from doxygen because of it. (E.g., it tries to apply the user `@`s with doxygen tags.)

Reply:
Thanks for point this out. The README.md as the main page for doxygen could be possible changed to something else or removed. I feel ambivalent about this however I can see that users mostly refer to the doxygen generated docs/html to inquire the API for the most part.